kiota_serialization_json/json_parse_node.py

```diff
@@ -300,11 +306,15 @@ def try_get_anything(self, value: Any) -> Any:
             return dict(map(lambda x: (x[0], self.try_get_anything(x[1])), value.items()))
         if isinstance(value, str):
             try:
+                if self.is_four_digit_number(value):
+                    print(value)
+                    return value
```

Comment:
does the value need to be casted? or parse to an integer?

Reply:
At this point, the value is already a json string, numeric types are dealt with in the get_int_value and get_float_value methods